core/tests/protractor_utils/forms.js

```diff
@@ -170,31 +174,34 @@ var ListEditor = function(elem) {
 var RealEditor = function(elem) {
   return {
     setValue: async function(value) {
-      await elem.element(by.tagName('input')).clear();
-      await elem.element(by.tagName('input')).sendKeys(value);
+      var realEditorInput = elem.element(by.tagName('input'));
+      await action.clear('Real Editor Input', realEditorInput);
+      await action.sendKeys('Real Editor Input', realEditorInput, value);
     }
   };
 };
 
 var RichTextEditor = async function(elem) {
   // Set focus in the RTE.
-  await waitFor.elementToBeClickable(elem.all(by.css('.oppia-rte')).first());
-  await (await elem.all(by.css('.oppia-rte')).first()).click();
+  var firstOfOppiaRte = await elem.all(by.css('.oppia-rte')).first();
+  await action.click('First of oppia rte', firstOfOppiaRte);
 
   var _appendContentText = async function(text) {
-    await (await elem.all(by.css('.oppia-rte')).first()).sendKeys(text);
+    var firstOfOppiaRte = await elem.all(by.css('.oppia-rte')).first();
+    await action.sendKeys(
+      'First of Oppia Rte', firstOfOppiaRte, text);
   };
   var _clickToolbarButton = async function(buttonName) {
-    await waitFor.elementToBeClickable(
-      elem.element(by.css('.' + buttonName)),
-      'Toolbar button takes too long to be clickable.');
-    await elem.element(by.css('.' + buttonName)).click();
+    var clickToolbarButton = elem.element(by.css('.' + buttonName));
+    await action.click(
+      `Click Toolbar Button: ${buttonName}`, clickToolbarButton);
   };
   var _clearContent = async function() {
     expect(
       await (await elem.all(by.css('.oppia-rte')).first()).isPresent()
     ).toBe(true);
-    await (await elem.all(by.css('.oppia-rte')).first()).clear();
+    var firstOfOppiaRte = await elem.all(by.css('.oppia-rte')).first();
+    await action.clear('First of oppia rte', firstOfOppiaRte);
```

Comment:
Here and elsewhere, please make the RTE element names clearer. For example:

```suggestion
    var firstOppiaRte = await elem.all(by.css('.oppia-rte')).first();
    await action.clear('First Rich Text Editor', firstOppiaRte);
```

Reply:
done!